library/std/src/sync/once_lock.rs

```diff
@@ -73,6 +43,56 @@ use crate::sync::Once;
 ///     Some(&12345),
 /// );
 /// ```
+///
+/// You can use `OnceLock` to implement a type that requires "append-only" logic:
+///
+/// ```
+/// #![feature(once_cell_try_insert)]
```

Comment:
could you write this without the unstable feature? I don't think it's great to have an example using unstable features

Reply:
yes, as it turns out!

Reply:
you're crazy